modules/types_protobuf/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+# =================================================================================================
+# Copyright (C) 2023-2024 HEPHAESTUS Contributors
+# =================================================================================================
+
+declare_module(
+  NAME types_protobuf
```

Comment:
internally we went for `types_proto` naming for all proto interface of native `types` module.

Reply:
Can you elaborate? `types_proto` is the name of the `photo` module, `types_protobuf` is the name of the module which contains the conversion `toProto` and `fromProto`. So there are
* `types`
* `types_proto` (somewhat hidden and usually only included in `types_protobuf`)
* `types_protobuf`

Reply:
Such change in convention shall be discussed further in our next call with @filippobrizzi and @floriantschopp .

Reply:
as mentioned, `from/toProto` and the protos themselves can be in the same module, which we call `module_proto`

Reply:
let's go for 
```suggestion
  NAME types_proto
```